tests/test_isrFunctions.py

```diff
@@ -0,0 +1,402 @@
+    print("%s %d %d\n" % (maskPlane, bitMask, numBit))
+    return numBit
+
+
+def imageMedianStd(image):
+    r"""Function to calculate median and std of image data.
+
+    Parameters
+    ----------
+    image : `lsst.afw.image.Image`
+        Image to measure statistics on.
+
+    Returns
+    -------
+    median : `float`
+        Image median.
+    std : `float`
+        Image stddev.
+    """
+    median = np.nanmedian(image.getArray())
+    std = np.nanstd(image.getArray())
+    print("ms: %f %f" % (median, std))
+    return (median, std)
+
+
+class IsrFunctionsCases(lsst.utils.tests.TestCase):
+    r"""Test functions for ISR produce expected outputs.
+    """
+    def setUp(self):
+        self.inputExp = isrMock.TrimmedRawMock().run()
+        self.mi = self.inputExp.getMaskedImage()
+
+    def tearDown(self):
+        pass
+
+    def test_transposeMaskedImage(self):
+        r"""Image transpose.
+
+        Expect height and width to be exchanged.
+        """
+        transposed = ipIsr.transposeMaskedImage(self.mi)
+        self.assertEqual(transposed.getImage().getBBox().getHeight(),
+                         self.mi.getImage().getBBox().getWidth())
+        self.assertEqual(transposed.getImage().getBBox().getWidth(),
+                         self.mi.getImage().getBBox().getHeight())
+
+    def test_interpolateDefectList(self):
+        r"""Defect interpolation.
+
+        Expect number of interpolated pixels to be non-zero.
+        """
+        defectList = isrMock.DefectMock().run()
+        assert len(defectList) > 0
+
+        for fallbackValue in (None, -999.0):
+            for haveMask in (True, False):
+                with self.subTest(fallbackValue=fallbackValue, haveMask=haveMask):
+                    if haveMask is False:
+                        if 'INTRP' in self.mi.getMask().getMaskPlaneDict():
+                            self.mi.getMask().removeAndClearMaskPlane('INTRP')
+                    else:
+                        if 'INTRP' not in self.mi.getMask().getMaskPlaneDict():
+                            self.mi.getMask().addMaskPlane('INTRP')
+                    numBit = maskCountPixels(self.mi, "INTRP")
+                    self.assertGreaterEqual(numBit, 0)
+
+    def test_transposeDefectList(self):
+        r"""Transpose defect list.
+
+        Expect bbox dimension values to flip.
+        """
+        defectList = isrMock.DefectMock().run()
+
+        transposed = ipIsr.transposeDefectList(defectList)
+
+        for d, t in zip(defectList, transposed):
+            assert d.getBBox().getDimensions().getX() == t.getBBox().getDimensions().getY()
+            assert d.getBBox().getDimensions().getY() == t.getBBox().getDimensions().getX()
+
+    def test_makeThresholdMask(self):
+        r"""Thresholding.
+
+        Expect list of defects to have elements.
+        """
+        defectList = ipIsr.makeThresholdMask(self.mi, 200, growFootprints=2, maskName='SAT')
+
+        self.assertGreater(len(defectList), 0)
```

Comment:
You should know the exact size of this `defectList` a priori, no? Similarly for the other `assertGreater` style tests below: Since you've constructed the test data, you should know the exact results.